app/controllers/subjects_controller.rb

```diff
@@ -1,6 +1,6 @@
 class SubjectsController < ApplicationController
   def index
-    search = Search.new nil, { facets: %w(subject), page_size: 0, facet_size: 500 }
+    search = Search.new nil, { facets: %w(subject), page_size: 0, facet_size: 240 }
```

Comment:
@mdellabitta Michael is right about caching.
@AudreyAltman  Keeping the facet size high doesn't help because the number of subjects is still limited to 10 pages of 30. 2000 facets greatly increases the size of the result that has to be serialized and transferred over the network and parsed by the Rails app, and that serialization / deserialization is not free.